plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestHiveGlueMetastore.java

```diff
@@ -179,16 +179,18 @@ public void testUpdateTableColumnStatistics()
     public void testUpdateTableColumnStatisticsEmptyOptionalFields()
             throws Exception
     {
-        // this test is not really meaningful for Glue, for numeric columns, min/max stat should always be available
-        // The only exception is when all rows are NULLs. Glue requires min/max to be set.
+        assertThatThrownBy(super::testUpdateTableColumnStatisticsEmptyOptionalFields)
+                .hasMessageContaining("Service: AWSGlue; Status Code: 500; Error Code: InternalServiceException;")
+                .isInstanceOf(TrinoException.class);
     }
 
     @Override
     public void testUpdatePartitionColumnStatisticsEmptyOptionalFields()
             throws Exception
     {
-        // this test is not really meaningful for Glue, for numeric columns, min/max stat should always be available
-        // The only exception is when all rows are NULLs. Glue requires min/max to be set.
```

Comment:
I'd keep the comment about why this fails. `Glue requires min/max to be set` or more accurately `Glue doesn't have any optional fields in statistics`.

But this doesn't match the documentation from AWS somehow. They mention min/max as `Required: No`.
If it's possible can you take a look at the response returned from AWS by enabling their debug logging as decribed at https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-logging.html#sdk-net-logging-request-response.